example_test.go

```diff
@@ -7,19 +7,17 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 or implied. See the License for the specific language governing permissions and
 limitations under the License. */
 
-package rs_test
+package rs
```

Comment:
There's a reason for using rs_test for examples: the example at https://pkg.go.dev/github.com/maruel/rs#example-NewDecoder clear show copy-pastable code with the import. Please revert.

Reply:
I see, I changed it to rs_test